plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/query/HudiQueryMode.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.hudi.query;
+
+public enum HudiQueryMode {
```

Comment:
It looks like this is not used anywhere (the only usage is hard-coded to `READ_OPTIMIZED`), so please remove for now. We can add it back later if needed.